app/src/main/java/com/example/tasky/itemdetail/domain/VisitorType.kt

```diff
@@ -0,0 +1,32 @@
+package com.example.tasky.itemdetail.domain
+
+import com.example.tasky.R
+
+sealed class VisitorType(
+    val name: Int,
+    val isSelected: Boolean
+) {
```

Comment:
This class only contains presentation related values (and is only needed there) and doesn't really represent a domain model, so it should go in presentation